### DISCUSSION TITLE
为什么前置插件不会一起安装呢？

### DISCUSSION THREAD

**Discussion by dongrentianyu** [tiddly-gittly/TiddlyWiki-CPL#229] [General]
比如我有一个插件，需要很多前置插件，但现在我安装这个插件时，只会提示某个插件没有，而不是随着这个插件一起安装前置插件。

![{E5B8E9C2-783B-4090-9D64-A95768758782}](https://github.com/user-attachments/assets/ee731f23-b947-420a-9fb9-b8dd0ebff14d)

而且这里的`$:/plugins/kookma/shiraz`插件我其实是有的。

![{3616EE22-7839-4050-A54E-F6A437D57FC2}](https://github.com/user-attachments/assets/89ccb3cb-c339-47d7-a081-da49584fe8e4)

在CPL里是这样填写的，我记得以前好像是没问题的来着。

**Comment by linonetwo** [comment]
@dongrentianyu 这里是类似太微的 `list` 字段的列表，不能加 `",` 空格隔开就行。如果包含空格就用 `[[ ]]` 包起来。

还有这个应该放到插件的 `plugin.info` 里吧，然后被 CPL 自动读取。

**Comment by linonetwo** [comment]
我现在不用 Aggregation 了，表格我都用 hyper-table 来展示，性能更高，而且 Aggregation 依赖有点多。当然缺点是暂时只读无法编辑。

**Comment by dongrentianyu** [comment]
好的，我看一下。我记得以前开发插件时，我就是这样写的。当时好像没问题。

**Reply by dongrentianyu** [reply]
应该是CPL里不不需要填写，只要在插件里填写就好了。在CPL里填写了反而会报错。
